packages/lesswrong/server/callbacks/postCallbacks.ts

```diff
@@ -160,11 +160,10 @@ getCollectionHooks("Posts").createAsync.add(async ({document}: CreateCallbackPro
 });
 
 getCollectionHooks("Posts").updateAsync.add(async function updatedPostMaybeTriggerReview ({document, oldDocument}: UpdateCallbackProperties<DbPost>) {
+  if (document.draft) return
 
   await triggerReviewIfNeeded(oldDocument.userId)
   
-  if (document.draft) return 
-  
```

Comment:
@Raemon My understanding is that draft posts don't trigger reviews, so I moved this check to the top of the function.